spec/requests/catalog_controller_spec.rb

```diff
@@ -49,7 +62,8 @@
 
     context 'can fetch data from other tenants' do
       xit 'cross-search-tenant can fetch all record in child tenants' do
-        allow(blacklight_solr_repository).to receive(:build_connection).and_return(RSolr.connect(url: 'http://solr:8983/solr/hydra-cross-search-tenant'))
+        connection = RSolr.connect(url: 'http://solr:8983/solr/hydra-cross-search-tenant')
```

Comment:
Might be best to put the setup for this in a `let`.

Reply:
Since this test is **xit**, it is better to leave it till i revisit the whole test.  Thanks.